internal/pkg/processor/config/config_test.go

```diff
@@ -129,7 +129,7 @@ func getTestData() []testData {
 					AheadBlocks:      16,
 					AheadBlockSize:   datasize.MB,
 					Gzip:             true,
-					GzipLevel:        2,
+					GzipLevel:        1,
```

Comment:
Default GZIP level has been switched from `2` to `1`, it is slightly faster with minimal impact to size.